TgeeT/Questioner
# Bug #163230116 fix specific meetup with validation and passing test

#### What does this PR do?

This PR is meant to fix the endpoint to get a specific meetup with validations and implementing the skipped test
#### Description of Task to be completed?

validate the data
Pass the failing tests

#### How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
2. git checkout to bg-fix-specific-meetup-163230116  branch
3. flask run the app and use postman to verify by trying to fetch unavailable data

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163230116
